RELEASE_NOTES.md

```diff
@@ -3,8 +3,38 @@ Release notes
 
 ### Next
 
-* Included support for compilation on Java 9 and 10. Compilation on Java 9 or 10 will produce a multi-release JAR that will run on Java 8, 9, and 10. This release was compiled on Java 8 and does not contain a multi-release JAR.
-* Changed metadata health check names to use the URI of the metadata they are trying to resolve
+#### Connect to Verify using only the VSP
+
+This release adds the ability to connect to GOV.UK Verify using only the Verify Service Provider (VSP). This means services can connect without needing to host a [Matching Service Adapter (MSA)](https://github.com/alphagov/verify-matching-service-adapter). If needed, services can then implement matching independently from their connection to GOV.UK Verify.
+
+Using only the VSP makes it easier to connect to GOV.UK Verify and reduces maintenance tasks once connected.
+
+When used alone, the VSP provides the required MSA functionality. It will:
+* implement security features to the same level as the MSA
+* handle both GOV.UK Verify and European identities
+* do Personal Identifier (PID) hashing
+
+If you are already connected to GOV.UK Verify and are running an MSA, you can upgrade to VSP 2.0.0 without needing to do any configuration changes.
+
+If you want to switch off your Matching Service Adapter and only use the Verify Service Provider to connect to Verify, [contact the GOV.UK Verify Team](https://www.verify.service.gov.uk/support/).
+
+For more information on connecting to GOV.UK Verify using only the VSP, see the [technical documentation](https://www.docs.verify.service.gov.uk).
+
```

Comment:
I think we need at a caveat here. Something like:
```suggestion
##### Caveats

It is not currently possible to test europeanIdentity with the VSP and the Compliance Tool. We recommend that you disable europeanIdentities support when you set the HUB_ENVIRONMENT configuration option to 'COMPLIANCE_TOOL'
```

Reply:
Following our chat, I won't be adding this comment in, because new users connecting with VSP 2.0.0 and above will not be using `HUB_ENVORONMENT` set to `COMPLIANCE_TOOL`. They'll be using the development command which automatically does the configuration and does not cause errors.
Follow up: remove `COMPLIANCE_TOOL` from the list of recommended values for `HUB_ENVIRONMENT` in the config.